code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -743,6 +743,15 @@
 	species_type_whitelist_typecache = list(/datum/species/diona)
 	sound = "sound/voice/dionatalk1.ogg"
 
+/datum/emote/living/carbon/human/diona_chirp
+	key = "chirp"
+	key_third_person = "chirps"
+	message = "chirps!"
+	message_param = "chirps at %t."
+	emote_type = EMOTE_AUDIBLE
```

Comment:
Might want to make it age_based, since most species emotes are and it adds a fun bit of flavor?

Reply:
Alright, sounds fine in testing so might as well.